### ISSUE TITLE
About coco pre-training

### ISSUE THREAD

**Issue by davidyang180** [dvlab-research/ECCV22-P3AFormer-Tracking-Objects-as-Pixel-wise-Distributions#12]
Hi! I found that there is pre-training on the coco dataset in the code of the project, and this part does not seem to be mentioned in the paper. It only says that the three mixed data sets of CrowdHuman, Cityperson and ETHZ are used. May I ask if adding the coco dataset pre-training to the model will lead to an additional advantage when comparing other tracking models? Because the vast majority of tracked models do not have this additional coco pretrained dataset.
